rules_python/whl.py

```diff
@@ -60,12 +61,12 @@ def metadata(self):
     with zipfile.ZipFile(self.path(), 'r') as whl:
       # first check for metadata.json
       try:
-        with whl.open(os.path.join(self._dist_info(), 'metadata.json')) as f:
+        with whl.open(posixpath.join(self._dist_info(), 'metadata.json')) as f:
```

Comment:
Is `posixpath` always available on Windows and other non-Posix platforms?  From the [zip specification](https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT) and the code for the `zipfile` module, it appears that literal forward slash '/' is the only thing that can be used here, so we could safely simplify this if we wanted to:

`with whl.open(self._dist_info() + '/metadata.json') as f`